v1/plugin/config_policy_test.go

```diff
@@ -34,63 +32,84 @@ func TestConfigPolicy(t *testing.T) {
 	tc := configPolicyTestCases()
 	Convey("Test Config Policy", t, func() {
 		for _, c := range tc {
-			Convey(fmt.Sprintf("Test Config Policy %+v", strings.Join(c.input.key, "/")), func() {
-				switch c.input.rule.(type) {
-				case stringRule:
-					srule := c.input.rule.(stringRule)
-					p.AddStringRule(c.input.key, srule)
+			Convey("Test Config Policy "+c.input.key, func() {
+				exist := false
+				switch c.input.opts.(type) {
+				case stringRuleOpt:
+					sopts := c.input.opts.(stringRuleOpt)
+					p.AddNewStringRule(c.input.ns, c.input.key, c.input.req, sopts)
 					r := newGetConfigPolicyReply(*p).GetStringPolicy()
-					Convey("Test ConfigPolicy stringRule", func() {
+					Convey("Test ConfigPolicy stringRule "+c.input.key, func() {
 						for _, v := range r {
 							for kk, vv := range v.GetRules() {
-								So(kk, ShouldEqual, srule.Key)
-								So(vv.Required, ShouldEqual, srule.Required)
-								So(vv.HasDefault, ShouldEqual, srule.HasDefault)
-								So(vv.Default, ShouldEqual, srule.Default)
+								if kk == c.output.key {
```

Comment:
`v.GetRules()` still return previous test cases of the same type. In this PR there are 2 string test cases, and 2 bool test cases. So we don't want to test the current rule with a former one.

But we still verify that the rule is there with the `exist` variable.